### PR TITLE
update link to oc compliance plugin

### PR DESCRIPTION
**Pull request by sluetze**
The old link was pointing to a fork which was 192 commits behind the original repo.

### CODE REVIEW

File: doc/tutorials/workshop/content/exercises/03-creating-your-first-scan.md [hunk 1]
```diff
@@ -480,7 +480,7 @@ Once you’re done with the extraction.
 
 > **NOTE**
 > 
-> Using the [`oc-compliance`](https://github.com/JAORMX/oc-compliance) plugin 
+> Using the [`oc-compliance`](https://github.com/openshift/oc-compliance) plugin
```
Comment (yuumasato) [inline doc/tutorials/workshop/content/exercises/03-creating-your-first-scan.md:483]: There is also another mention of `oc-compliance` on line 302, would you mind fixing it there too?

**Review by yuumasato — COMMENTED** [review]
Nice catch, and thanks for the patch!

**Comment by rhmdnd** [comment]
Adding necessary labels since this is fixing an incorrect reference in documentation.

**Pull request merged**